pkg/instancetype/instancetype.go

```diff
@@ -1201,3 +1202,10 @@ func applyClockPreferences(preferenceSpec *instancetypev1beta1.VirtualMachinePre
 		vmiSpec.Domain.Clock.Timer = preferenceSpec.Clock.PreferredTimer.DeepCopy()
 	}
 }
+
+func applyTerminationGracePeriodSeconds(preferenceSpec *instancetypev1beta1.VirtualMachinePreferenceSpec, vmiSpec *virtv1.VirtualMachineInstanceSpec) {
+	if preferenceSpec.PreferredTerminationGracePeriodSeconds == nil {
+		return
+	}
+	vmiSpec.TerminationGracePeriodSeconds = pointer.Int64(*preferenceSpec.PreferredTerminationGracePeriodSeconds)
```

Comment:
What if `vmiSpec.TerminationGracePeriodSeconds != nil`? This would override the value set by the user? (no longer a preference)